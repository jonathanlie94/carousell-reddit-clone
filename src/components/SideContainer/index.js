import React, { Component } from 'react';
import styled from 'styled-components';
import { theme } from 'ui';

const Wrapper = styled.div`
  background-color: ${theme.colors.grey};
  color: ${theme.colors.black};
  min-width: 320px;
  height: auto;
  padding: 12px 12px 12px 12px;
`;

class SideContainer extends Component {
  render() {
    return (
      <Wrapper>
        {this.props.children}
      </Wrapper>
    );
  }
}

export default SideContainer;
