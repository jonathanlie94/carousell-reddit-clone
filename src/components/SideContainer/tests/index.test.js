import React from 'react';
import { shallow } from 'enzyme';
import SideContainer from '../';

describe('<SideContainer />', () => {
  it('should render children when passed in', () => {
    const renderedComponent = shallow(
      <SideContainer>
        <div className="unique-class-name"></div>
      </SideContainer>
    );
    expect(renderedComponent.contains(<div className="unique-class-name"></div>)).toEqual(true);
  });
});
