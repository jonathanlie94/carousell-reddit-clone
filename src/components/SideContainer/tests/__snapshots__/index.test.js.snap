// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SideContainer /> should match previous snapshot 1`] = `<styled.div />`;
